.github/workflows/install-and-run-spsh-loadtest.yml

```diff
@@ -16,14 +16,14 @@ on:
         type: string 
         required: false
         default: "debug"
-      scenario:
+      scenario-list:
```

Comment:
Is '-' allowed in GHA input names?

Reply:
https://github.com/dBildungsplattform/schulportal-load-tests/actions/runs/11501472986/job/32014138021 seems to be okay here , but new error seems to exist

Reply:
Sometimes YAML doesn't like dashes but if it works it's fine